packages/fxa-shared/nestjs/sentry/reporting.ts

```diff
@@ -122,6 +122,10 @@ export function reportRequestException(
   excContexts: ExtraContext[] = [],
   request?: Request
 ) {
+  // Don't report already reported exceptions
```

Comment:
This is needed because the Sentry interceptor *and* the lifecycle hook error reporter are triggered by the same mutation/query level error.